src/sentry.ts

```diff
@@ -103,7 +104,7 @@ export default class SentryService {
       };
 
       for (const flag of non_default_flags) {
-        sentry_tags[`flag.${flag}`] = command_class.flags[flag].sensitive ? 'Filtered' : flags[flag];
+        sentry_tags[`flag.${flag}`] = command_class.flags?.[flag]?.sensitive ? 'Filtered' : flags?.[flag];
```

Comment:
Is this change necessary with the `present_flags_keys` filtering above? It seems like that's guaranteeing that `command_class.flags[flag]` will always exist. Also can `flags` ever be undefined? `flags?[flag]` seems like it shouldn't be necessary, I wouldn't expect parse to return undefined flags

Reply:
Tested a blank `architect.yml` without optional chaining, and it runs OK with the filtering method above using `present_flags_keys`. Updated to remove the chaining